sdk/communication/Azure.Communication.Sms/src/Models/SmsModelFactory.cs

```diff
@@ -1,18 +1,22 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using System.Collections.Generic;
```

Comment:
is the second `using` necessary?

Reply:
I believe so, see the other comment for: https://stackoverflow.com/questions/14060379/import-all-subclasses-like-java-but-in-c-sharp